.vscode/settings.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Did you meant to commit this file? We haven't been commiting vscode settings to our projects thus far and I'm not sure which of these settings make sense for everyone working on this projects vs just making sense for your personal workflow? Maybe we should add a `.vscode` to the gitignore list?

Reply:
Thank you great catch. I'm removing these files now. Not sure how they got in there.